packages/modules/pt/src/pages/citizen/Create/Response.js

```diff
@@ -0,0 +1,162 @@
+import { Card, Banner, CardText, SubmitBar, LinkButton } from "@egovernments/digit-ui-react-components";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Loader } from "@egovernments/digit-ui-react-components";
+//import getPDFData from "../../../getPDFData";
+
+const GetActionMessage = () => {
+  const { t } = useTranslation();
+  return t("CS_FILE_DESLUDGING_APPLICATION_SUCCESS");
+};
+
+const BannerPicker = (props) => {
+  return (
+    <Banner
+      message={GetActionMessage()}
+      applicationNumber={props.data?.Properties[0].propertyId}
+      info={props.t("CS_FILE_DESLUDGING_APPLICATION_NO")}
+      successful={props.isSuccess}
+    />
+  );
+};
+
+const Response = ({ data, onSuccess }) => {
+  const { t } = useTranslation();
+  const tenantId = Digit.ULBService.getCurrentTenantId();
+  let mutation = {};
+
+  useEffect(() => {
+    try {
+      const { subtype, pitDetail, address, pitType, source, ownershipCategory, owners } = data;
+
+      const loc = address?.locality.code;
+      const formdata = {
+        Property: {
+          tenantId: address?.city.code,
+          address: {
+            city: address?.city.name,
+            doorNo: address?.doorNo,
+            buildingName: "NA",
+            locality: {
+              code:
+                address?.city.code === "pb.amritsar"
+                  ? loc.substring(loc.length - 5, loc.length)
+                  : address?.city.code === "pb.jalandhar"
+                  ? loc.substring(loc.length - 6, loc.length)
+                  : loc.substring(loc.length - 5, loc.length),
+              area: address?.locality.name,
+            },
+          },
+          usageCategoryMinor: null,
+          units: [
+            {
+              occupancyType: "SELFOCCUPIED",
+              floorNo: "0",
+              constructionDetail: {
+                builtUpArea: 16.67,
+              },
+              tenantId: address?.city.code,
+              usageCategory: "RESIDENTIAL",
+            },
+          ],
+          usageCategoryMajor: "RESIDENTIAL",
+          landArea: "2000",
+          propertyType: "BUILTUP.SHAREDPROPERTY",
+          noOfFloors: 1,
+          ownershipCategory: "INDIVIDUAL.SINGLEOWNER",
+          owners: [
+            {
+              name: owners[0]?.name,
+              mobileNumber: "7838038768",
+              fatherOrHusbandName: owners[0]?.fatherOrHusbandName,
+              emailId: "tutul.tulika@gmail.com",
+              permanentAddress: owners[0]?.permanentAddress,
+              relationship: "FATHER",
+              ownerType: "NONE",
+              gender: "Male",
+              isCorrespondenceAddress: null,
+            },
+          ],
+          additionalDetails: {
+            inflammable: false,
+            heightAbove36Feet: false,
+          },
+          source: "MUNICIPAL_RECORDS",
+          channel: "CFC_COUNTER",
+          documents: [
```

Comment:
@Tulika-eGov / @vamshikrishnakole-wtt-egov  Can you map uploaded Documents instead of hardcoded documents